lib/redux/actions.dart

```diff
@@ -37,7 +37,8 @@ class FetchOrganizationsSuccessAction {
 }
 
 class FetchOrganizationsFailureAction {
-  FetchOrganizationsFailureAction();
+  FetchOrganizationsFailureAction(this.error);
+  final Error error;
```

Comment:
```suggestion
  final Error error;
  FetchOrganizationsFailureAction(this.error);
```

Reply:
also the other classes, I'd keep the fields before ctor

Reply:
Seems to be the Dart Lang preference to sort the constructor first. I also get this as a warning by the linter. Would rather stay in this style. https://dart-lang.github.io/linter/lints/sort_constructors_first.html